atom/browser/api/atom_api_window.cc

```diff
@@ -83,7 +83,7 @@ Window::Window(v8::Isolate* isolate, v8::Local<v8::Object> wrapper,
   v8::Local<v8::Value> transparent;
   if (options.Get("transparent", &transparent))
     web_preferences.Set("transparent", transparent);
-
```

Comment:
The error on your build is coming from a linting issue with this line

> atom/browser/api/atom_api_window.cc:86:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

You need to get rid of this whitespace 👍